site/sidebar.md

```diff
@@ -37,6 +37,9 @@
 	- [8 Package Orchestration](book/08-package-orchestration/)
 		- [8.1 System Requirements](book/08-package-orchestration/01-system-requirements.md)
 		- [8.2 Quickstart](book/08-package-orchestration/02-quickstart.md)
+		- [8.3 Registering a Repository](book/08-package-orchestration/03-registering-a-repository.md)
```

Comment:
Are these changes related to this PR?

Reply:
No, I think this is just from a previous PR that didn't run the script to generate the sidebar. I thought I'd just keep it here to get it fixed, but I can move it into a separate PR if you prefer that.